views/templates/admin/addons-suggestion.tpl

```diff
@@ -28,22 +28,21 @@
   </div>
   <div class="suggestion-category-details">
     <div>
-      {l s='To go further:' d='Modules.Productcomments.Shop'}
+      {l s='To go further:' d='Admin.Modules.Feature'}
     </div>
     <div class="category-label">
-      {l s='Customer reviews' d='Modules.Productcomments.Shop'}
+      {$addons_category.name}
     </div>
     <div class="marketplace-label">
-      {l s='Addons Marketplace' d='Modules.Productcomments.Shop'}
+      {l s='Addons Marketplace' d='Admin.Modules.Feature'}
```

Comment:
_Admin.Global_ please, it is a very common string that will sure be reused elsewhere in the software. ;-)

Reply:
Alright, done